catalog/dags/common/licenses/licenses.py

```diff
@@ -23,11 +23,26 @@ class InvalidLicenseURLException(Exception):
     pass
 
 
-LicenseInfo = namedtuple("LicenseInfo", ["license", "version", "url", "raw_url"])
+class LicenseInfo(NamedTuple):
+    license: str
+    version: str
+    url: str
+    raw_url: str | None = None
+
+
+# SMK sometimes uses incorrect URL for licenses
+special_cases = {
+    "https://creativecommons.org/share-your-work/public-domain/cc0/": "https://creativecommons.org/publicdomain/zero/1.0/",
+    "https://creativecommons.org/share-your-work/public-domain/pdm/": "https://creativecommons.org/publicdomain/mark/1.0/",
+}
```

Comment:
I actually added these here instead of a separate PR because after adding checks for falso LicenseInfo, smk tests started failin. You can see that some of the smk test jsons have a 'share' URL

Reply:
Added a unit test to `test_licenses` 👍